pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["calle <callebtc@protonmail.com>"]
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8.1"
```

Comment:
Probably fine, we're early enough that it should be OK to drop 3.7 without upsetting anyone 😆  - ping @callebtc just to check

Reply:
i talked to calle about it :) i think its fine